kafka-client/src/test/java/dev/responsive/kafka/async/AsyncProcessorIntegrationTest.java

```diff
@@ -177,91 +199,55 @@ public void shouldProcessEventsInOrderByKey() throws Exception {
     assertThat(processed.get(), equalTo(numProcessedRecords));
     assertThat(latestValues, equalTo(finalOutputRecords));
   }
-  
-  private static class UserFixedKeyProcessor implements FixedKeyProcessor<String, String, String> {
```

Comment:
There's a small change to the output format (for obvious reasons) but other than that nothing changed about the way this processor works. I just extracted it out to a standalone class so I can use the framework to plug in stateful processors more easily across integration tests